packages/module/src/TagCount/TagCount.tsx

```diff
@@ -12,7 +12,8 @@ const useStyles = createUseStyles({
   },
 
   tagText: {
-    marginLeft: 10
+    marginLeft: 6,
```

Comment:
this one could be replaced with a variable as well
the closest is probably "--pf-v5-global--spacer--sm" 0.5rem. 
@kaylachumley according to PF rules, do you think we should go with hardcoded 6px or that variable? Thank you